### PR TITLE
feat: fix docker-compose files for bff and db

### PR DESCRIPTION
**Pull request by anversa-pro**
## Issue ticket number and link
https://trello.com/c/R0qN4dDw

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [x] :sparkles: Feature
- [x] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Dockerized BFF and database (DB) services not connecting correctly.

## What is the new behavior?

- Updated docker-compose files to fix issues with the Dockerized BFF and DB services.
- Added a Docker volume for data persistence to avoid data loss when containers are restarted.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by FelipeLosada1996 — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -3,15 +3,17 @@ version: '3.7'
...
   mongodb:
+    container_name: mongodb
     image: mongo:6.0.6
     ports:
       - '27017:27017'
     environment:
-      - MONGO_INITDB_ROOT_USERNAME=mongoDev
-      - MONGO_INITDB_ROOT_PASSWORD=Passw0rd
-      - MONGO_INITDB_DATABASE=mongodb
+      - MONGO_INITDB_ROOT_USERNAME=$MONGO_INITDB_ROOT_USERNAME
```
Comment (jose-sal-y-rosas) [inline docker-compose.yml:11]: I think the content of this variable is missing? Where are they define?

File: docker-compose.local.yml [hunk 2]
```diff
@@ -29,7 +13,9 @@ services:
     depends_on:
       - mongodb
     environment:
-      - MONGODB_URI='mongodb://mongoDev:Passw0rd@mongo:27017/?authSource=admin'
-      - DEFAULT_DB=mongodb
+      - MONGODB_URI=$MONGODB_URI_COMPOSE
```
Comment (jose-sal-y-rosas) [inline docker-compose.local.yml:16]: Same here, what is the value for this variable and where is define?

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: docker-compose.yml (hunk 1 above)
Comment (anversa-pro) [reply docker-compose.yml:11]: I removed sensitive data and defined it in the .env and .env.local file.

File: docker-compose.local.yml (hunk 2 above)
Comment (anversa-pro) [reply docker-compose.local.yml]: I removed sensitive data and defined it in the .env and .env.local file.

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**